src/afl-fuzz-queue.c

```diff
@@ -1007,10 +1007,16 @@ u32 calculate_score(afl_state_t *afl, struct queue_entry *q) {
       break;
 
     case LIN:
+      // Don't modify perf_score for unfuzzed seeds
+      if (!q->fuzz_level) break;
+
       factor = q->fuzz_level / (afl->n_fuzz[q->n_fuzz_entry] + 1);
       break;
 
     case QUAD:
+      // Don't modify perf_score for unfuzzed seeds
+      if (!q->fuzz_level) break;
+
```

Comment:
why only for for lin and quad, why not for all?

Reply:
Factorization is only performed for exploit, fast, coe, lin, and quad schedules. And fuzz_level, if used, checked everywhere except lin and quad.

Reply:
I don't know how other power schedules should handle seeds with fuzz_level=0, but for lin and quad perf_score will be turned to a null in this case. And when combined with MOpt score would not be bounded to a 1 at https://github.com/AFLplusplus/AFLplusplus/blob/f01bf77604c7d5b2ee12a44cbb3dbd615ad9565e/src/afl-fuzz-queue.c#L1058